specials/ladybug/build.xml

```diff
@@ -41,14 +41,14 @@
 		<property name="pom.xml" location="${basedir}/../../../${ff.dir}/ladybug/pom.xml" if:true="${test.with.iaf}"/>
 		<condition property="ladybug.version.ok">
 			<or>
-				<resourcecontains resource="${pom.xml}" substring="&lt;ladybug.version&gt;${ladybug.pom.xml.project.version}&lt;/ladybug.version&gt;"/>
+				<resourcecontains resource="${pom.xml}" substring="&lt;ladybug.version&gt;${ladybug.pom.xml.project.properties.revision}&lt;/ladybug.version&gt;"/>
 				<and>
 					<equals arg1="${test.with.test.webapp}" arg2="false"/>
 					<equals arg1="${test.with.iaf}" arg2="false"/>
 				</and>
 			</or>
 		</condition>
-		<fail message="Check ladybug version failed:&#10;&#10;MANUAL ACTION REQUIRED: Set version of artifact ibis-ladybug to ${ladybug.pom.xml.project.version} in ${pom.xml} " unless:true="${ladybug.version.ok}"/>
+		<fail message="Check ladybug version failed:&#10;&#10;MANUAL ACTION REQUIRED: Set version of artifact ibis-ladybug to ${ladybug.pom.xml.project.properties.revision} in ${pom.xml} " unless:true="${ladybug.version.ok}"/>
```

Comment:
Het is niet meer `ibis-ladybug` maar gewoon `ladybug`. Misschien kun je die verbetering gelijk meenemen, als is het niet de scope van je fix.

Reply:
Daar heb je groot gelijk in. Zal het toevoegen aan de PR